src/api/app/helpers/flag_helper.rb

```diff
@@ -130,13 +130,13 @@ def set_repository_by_product(flag, status, product_name, patchlevel = nil)
   def enabled_for?(flag_type, repo, arch)
     state = find_flag_state(flag_type, repo, arch)
     logger.debug "enabled_for #{flag_type} repo:#{repo} arch:#{arch} state:#{state}"
-    return state.to_sym == :enable ? true : false
+    state.to_sym == :enable ? true : false
```

Comment:
yes, I didn't change it because I considered that was unrelated to removing the return. I'll add it another commit for that!